schema/extension.go

```diff
@@ -20,6 +20,9 @@ type (
 			Password struct {
 				Identifier bool `json:"identifier"`
 			} `json:"password"`
+			Sms struct {
```

Comment:
Convention says that abbreviations are upper case in golang:

```suggestion
			SMS struct {
```

Reply:
Renamed `SMS` to `Code`